documentation/book/topic-operator.adoc

```diff
@@ -235,7 +235,21 @@ Default `6`.
 – The level for printing logging messages. 
 The value can be set to: `ERROR`, `WARNING`, `INFO`, `DEBUG` and `TRACE`. 
 Default `INFO`.
-
+* `STRIMZI_TLS_ENABLED`
+– For enabling the TLS support so encrypting the communication with Kafka brokers and Zookeeper nodes.
```

Comment:
I think it should be "For enabling the TLS support to encrypt the communication ...."